src/MusicalScore/Graphical/VexFlow/VexFlowConverter.ts

```diff
@@ -543,10 +541,10 @@ export class VexFlowConverter {
             // add Tremolo strokes (only single note tremolos for now, Vexflow doesn't have beams for two-note tremolos yet)
             const tremoloStrokes: number = notes[i].sourceNote.TremoloStrokes;
             if (tremoloStrokes > 0) {
-                const tremolo: Vex.Flow.Tremolo = new Vex.Flow.Tremolo(tremoloStrokes);
-                (tremolo as any).extra_stroke_scale = rules.TremoloStrokeScale;
-                (tremolo as any).y_spacing_scale = rules.TremoloYSpacingScale;
-                vfnote.addModifier(i, tremolo);
+                const tremolo: Tremolo = new Tremolo(tremoloStrokes);
+                // rvilarl tremolo.extra_stroke_scale = rules.TremoloStrokeScale;
+                // rvilarl tremolo.y_spacing_scale = rules.TremoloYSpacingScale;
```

Comment:
tremolo does not have these attributes.

Reply:
It does in our patched version in `VexFlowPatch/`. We'll need to adjust our VexFlowPatch files in any case.

Reply:
I cannot revert, it will not compile.